packages/api/src/middlewares/withFilters.ts

```diff
@@ -155,7 +155,7 @@ export const withFilters = t.middleware(({ next, input = {} }) => {
       filters.transactionAddresses = [];
       if (from) {
         filters.transactionAddresses.push({
-          fromId: from,
+          fromId: { in: from.split(",") },
```

Comment:
Let's add tests to check that this filter works as it suppose to in the following cases:

- when providing multiple values
- when providing empty values (e.g. `",,,,"`, `"value1,,value3"`, etc)

Reply:
Added 👍